rids/observations.py

```diff
@@ -12,31 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-Tests for the network_capture library.
-"""
 
+from dataclasses import dataclass
+import ipaddress
+from typing import Union
 
-import io
-import pytest
 
-from rids import network_capture
+IpType = Union[ipaddress.IPv4Address, ipaddress.IPv6Address]
```

Comment:
Name it `IpAddress`. Having `Type` in names is usually bad practice as they are not informative.
```suggestion
IpAddress = Union[ipaddress.IPv4Address, ipaddress.IPv6Address]
```

Reply:
It is a type representation not a class, though, so I prefer to include the Type in this name.  If it were something you could instantiate from, I would agree with you that Type is extra, but in this case I think it is informative of it being a sum type not a class name.  Can we compromise on IpAddressType?

Reply:
I'll leave that up to you.

The issue was not whether to add `Address`, but whether to include `Type`. None of the standard type annotations include `Type`, for example: https://docs.python.org/3/library/typing.html. With that naming convention you'd have to say `IpType = UnionType[ipaddress.IPv4AddressType, ipaddress.IPv6AddressType]` instead.

Reply:
After looking at a lot of Union types in core Python and other libs, I've removed the Type from this type variable name.